packages/react/src/components/Hero/Hero.tsx

```diff
@@ -58,7 +58,9 @@ const Hero: FC<HeroProps> = ({
 
   return (
     <div className={heroClasses}>
-      {image ? <HeroImage {...image} /> : <div className="hero--figure" />}
+      <div className="hero--figure">
```

Comment:
hero--background would be a better name

Reply:
Sure

Reply:
Will do, in that case will have to make the change in twig as well.